fastlane/lanes/build.rb

```diff
@@ -266,6 +266,22 @@
     )
   end
 
+  lane :build_for_app_store_connect_reader do
+    sentry_check_cli_installed
+
+    update_certs_and_profiles_app_store_reader
```

Comment:
Like the lane to fetch the cert and provisioning profile, this lane has "reader" as the suffix. I'll follow up at some point to adjust the names of all lanes so they are consistent and tab-completion friendly.

With that I mean

- `build_for_...`
- `build_for_firebase_...
- `build_for_app_store_connect_...`
- `build_for_app_store_connect_jetpack`
- `build_for_app_store_connect_wordpress`
- `build_for_app_store_connect_reader`

But I need to take renames slowly because I need to check usages in Rv2.